projects/admin/src/app/acquisition/routes/receipt-lines-route.ts

```diff
@@ -76,4 +78,19 @@ export class ReceiptLinesRoute extends BaseRoute implements RouteInterface {
     };
     return field;
   }
+
+  /**
+   * Pre update record
+   * @param data - Record data
+   * @returns record data
+   */
+   private _preUpdateRecord(data: any): any {
+    const fieldsToRemoved = ['is_current_budget'];
+    fieldsToRemoved.forEach(key => {
+      if (key in data) {
+        delete data[key];
+      }
+    });
+    return data;
```

Comment:
Suggestion : As this method seems repetitive for some routes, could be useful to place it into `BaseRoute` with `fieldsToRemoved` as required parameter.